packages/sync/src/utils/buildSyncLookupUtils.ts

```diff
@@ -0,0 +1,8 @@
+import { DatabaseModel } from '@tupaia/database';
+
+import { buildSyncLookupSelect } from './buildSyncLookupSelect';
+
+export const buildSyncLookupSurveyProjectIdSelect = async (model: DatabaseModel) =>
```

Comment:
This is also quite small, and maybe confusing when it is so tightly coupled to a context that it is far away from. What do you think?